src/test/java/org/javarosa/xform/parse/RepeatGroupsRelevanceTest.java

```diff
@@ -0,0 +1,39 @@
+package org.javarosa.xform.parse;
+
+import org.javarosa.core.model.FormDef;
+import org.javarosa.core.test.FormParseInit;
+import org.javarosa.form.api.FormEntryModel;
+import org.junit.Test;
+
+import static org.javarosa.test.utils.ResourcePathHelper.r;
+import static org.junit.Assert.assertFalse;
+
+public class RepeatGroupsRelevanceTest {
+
+    @Test
+    public void isRepeatRelevantTest() {
+        FormParseInit fpi = new FormParseInit(r("repeat_groups_relevance.xml"));
```

Comment:
One thing I did in #581 is split this form and test into several smaller ones at https://github.com/getodk/javarosa/pull/581/files#diff-02a3eeffc5c34a1d10f7680159593ac9R62.

I just want to point out that this is a "do as I say, not as I do" case because I did the same kind of monolithic test at https://github.com/getodk/javarosa/pull/578/commits/c954d0bc29e7c521702a42670de856a92859922c#diff-c9232740659aac312cd026fe65a452f8R24 and @seadowg had to tell me to split it up. I think the split has to be a later pass once we know what's going on and it's hard to remember to do that!